app/src/main/java/io/horizontalsystems/bankwallet/modules/send/submodules/amount/SendAmountPresenter.kt

```diff
@@ -10,21 +11,19 @@ import java.math.RoundingMode
 
 
 class SendAmountPresenter(
-        private val interactor: SendAmountModule.IInteractor,
+        val view: SendAmountModule.View,
+        private val interactor: SendAmountModule.Interactor,
         private val presenterHelper: SendAmountPresenterHelper,
         private val coin: Coin,
         private val baseCurrency: Currency)
-    : SendAmountModule.IViewDelegate, SendAmountModule.IInteractorDelegate, SendAmountModule.IAmountModule {
+    : ViewModel(), SendAmountModule.ViewDelegate, SendAmountModule.InteractorDelegate, SendAmountModule.AmountModule {
 
-    var view: SendAmountModule.IView? = null
```

Comment:
Since field view is not optional anymore, using ? operator is redundant now (eg. onViewDidLoad)